dplython/dplython.py

```diff
@@ -339,10 +344,21 @@ def f(df):
   return f
 
 
-@ApplyToDataframe
-def head(*args, **kwargs):
-  """Returns first n rows"""
-  return lambda df: df.head(*args, **kwargs)
+class head(Verb):
+  """returns first n rows; maintains grouping if present"""
+
+  __name__ = "head"
+
+  def __call__(self, df):
+    n = 5 if not self.args else self.args[0]
+    if df._grouped_on:
+      return (df >> ungroup()).head(n).regroup(df._grouped_on)
```

Comment:
I made a minor change here. You can look at the code I committed, but I think it's better to pass args and kwargs to `head` here. That way we always default to the pandas behavior, and if pandas changes the defaults for their `head` method, we will change too automatically along with it.